fft/src/KokkosFFT_Cuda_transform.hpp

```diff
@@ -10,45 +10,47 @@
 
 namespace KokkosFFT {
 namespace Impl {
-template <typename... Args>
-inline void exec_plan(cufftHandle& plan, cufftReal* idata, cufftComplex* odata,
-                      int /*direction*/, Args...) {
-  cufftResult cufft_rt = cufftExecR2C(plan, idata, odata);
+template <typename ScopedPlanType>
+inline void exec_plan(ScopedPlanType& scoped_plan, cufftReal* idata,
```

Comment:
Can you pass the `scoped_plan` by `const&` ?

Reply:
Modified accordingly